simpledotorg/simple-server
# Create EstimatedPopulation model

**Story card:** [ch6171](https://app.shortcut.com/simpledotorg/story/6171/create-a-new-estimatedpopulation-model)

## Because

This PR allows Dashboard admins to set a district's estimated hypertensive population value. It adds a new "Estimated hypertensive population" field to the "Edit district" admin page and creates/updates the respective models/tests to enable this.

## This addresses

* Creates a new `EstimatedPopulation` model and respective tests
* Updates the `Region` and `FacilityGroup` models
* Adds a new "Estimated hypertensive population" field to the `.../facility_groups/_form.html.erb` view

## Test instructions

1. Go to the "Facilities" admin page: `http://localhost:3000/admin/facilities`
2. Scroll through the page and verify that all cards have an "Estimated hypertensive population:" subtitle AND that the subtitle shows either a comma-separated numeric value OR a "+ Add population" link
3. Click the "Edit district" button for one of the facility groups and edit the "Estimated hypertensive population" value
4. Save the changes and verify that the facility group's population value has been updated in the "Facilities" admin page
5. Find a different facility group and click the "Edit district" button. This time, delete the "Estimated hypertensive population" value and save the changes
6. Go back to the "Facilities" admin page and verify the facility group card shows the "+ Add population" link
7. Click the "+ Add population" link and set a population value
8. Go back to the "Facilities" admin page and verify the value is displayed in the facility group card
